hack/label-worker-rt.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
who calls this? only humans or also the operator proper? I guess the former, asking to be sure.

Reply:
we use a `MachineConfig` for installing this script and a systemd unit, and systemd will execute the script then.

Reply:
oh sorry, just noticed I was looking at the wrong script
I called this one manually on test clusters